osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/services/ConfigurationService.java

```diff
@@ -246,7 +257,7 @@ public String enqueueSetRandomisationSettingsRequest(@Identification final Strin
      */
     private String enqueueAndSendRequest(final String organisationIdentification, final String deviceIdentification,
```

Comment:
In order to reduce the number of parameters consider splitting this method up into 3 methods for:
- checking authorizations
- creating metadata (incl. correlation uid)
- sending message

Reply:
Changed